contents/handbook/company/communication.md

```diff
@@ -104,6 +104,16 @@ GitHub Issues are useful when there isn't a specific code change that is being p
 
 However, it is still important to maintain focus when opening issues by defining a single specific topic of discussion as well as defining the desired outcome that would result in the resolution of the issue. The point is to not keep issues open-ended and to prevent issues from going stale due to lack of resolution. For example, a team member may open an issue to track the progress of a blog post with associated to-do items that need to be completed by a certain date (e.g. first draft, peer review, publish). Once the specific items are completed, the issue can successfully be closed.
 
+### Product definitions & decisions
+
+It's best practice to get feedback from other members of the team when making a product decision or defining a complex problem that needs to be solved. This may start with an Issue, but it's essential that these do not spiral out of control - as soon as an issue has more than 5 comments, it becomes difficult for someone to "catch up". To avoid this problem, when we're documenting a decision or a definition for a project or project, we should start with a Pull Request.
```

Comment:
:thinking: What do you call about explicitly calling these documents RFCs? 

We don't need much explicit structure besides tagging it with a date - it'll be out of date as soon as published anyways.

We can then even have these under posthog.com by default, with company/product-internal things being under a private repo if warranted.

Reply:
Created an RFC template in https://github.com/PostHog/meta/pull/19